README.md

```
@@ -1,54 +1,114 @@
+    * A swipe down on the Wifi page triggers a manual Wifi refresh which directly asks the Android
+      system to rescan to receive the newest and most precise Signal Strength values.
+* Dual-SIM Support
+    * When the app detects 2 usable SIM cards, it switches the view to not only display a button for
+      Cellular but for SIM1 and SIM2, so the user can choose which SIM to use and compare.
+* Single-SIM Detection
+    * When the app finds no usable SIM cards, it automatically switches the view to Wifi and
+      disables the Cellular page.
+
+**Screenshots**
+<p>
+  <img src="images/Screenshot_Welcomepage.jpg" width="250" />
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)